doc/nrf/known_issues.rst

```diff
@@ -752,7 +753,7 @@ Closing sockets
 Multiprotocol Service Layer (MPSL)
 ==================================
 
-.. rst-class:: v1-5-0 v1-4-2 v1-4-1
+.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1
 
 DRGN-15223: `CONFIG_SYSTEM_CLOCK_NO_WAIT` is not supported for nRF5340
```

Comment:
@hurricaneFromMoscow , DRGN-15223 is fixed for NCS 1.5.1, right?

Reply:
Yes, the fix is present on the v1.5-branch

Reply:
Updated